packages/preact/src/index.ts

```diff
@@ -388,10 +388,22 @@ export function useComputed<T>(compute: () => T) {
 	return useMemo(() => computed<T>(() => $compute.current()), []);
 }
 
+const HAS_RAF = typeof requestAnimationFrame == "function";
+function afterNextFrame(callback: () => void) {
+	let raf: number | undefined;
+
+	const done = () => {
+		clearTimeout(timeout);
+		if (raf != null) cancelAnimationFrame(raf);
+		callback();
+	};
+
+	const timeout = setTimeout(done, 100);
+	if (HAS_RAF) raf = requestAnimationFrame(done);
+}
```

Comment:
I just realized we don't need `HAS_RAF` at all here, since this function is only used when `typeof requestAnimationFrame !== 'undefined'` due to line 405.

```suggestion
function afterNextFrame(callback: () => void) {
	const done = () => {
		clearTimeout(timeout);
		cancelAnimationFrame(raf);
		callback();
	};

	const timeout = setTimeout(done, 100);
	const raf = requestAnimationFrame(done)
}
```